documentation/modules/mtv-aio-buffer.adoc

```diff
@@ -0,0 +1,253 @@
+
+.Verifying migration logs
+
+* Run a migration plan and check the logs of the migration pod. Confirm that the AIO buffer settings are passed as parameters, particularly the `--vddk-config value`.
++
+For example:
++
+[source,termina]
+----
+exec: /usr/bin/virt-v2v … --vddk-config /mnt/extra-v2v-conf/input.conf
+----
+
++
+Sample log excerpt:
++
+[source,termina]
+----
+Buffer size calc for 16 value:
+(16 * 64 * 1024 = 1048576)
+nbdkit: vddk[1]: debug: [NFC VERBOSE] NfcAio_OpenSession:
+Opening an AIO session.
+nbdkit: vddk[1]: debug: [NFC INFO] NfcAioInitSession:
+Disabling
+read-ahead buffer since the AIO buffer size of 1048576 is >=
+the read-ahead buffer size of 65536. Explicitly setting flag
+'`NFC_AIO_SESSION_NO_NET_READ_AHEAD`'
+nbdkit: vddk[1]: debug: [NFC VERBOSE] NfcAioInitSession: AIO Buffer Size is 1048576
+nbdkit: vddk[1]: debug: [NFC VERBOSE] NfcAioInitSession: AIO Buffer
+Count is 4
+----
++
+[NOTE]
+====
+The above logs were when using `debug_level = 4`
+====
+
+.Inspecting ConfigMap values Content are in the Migration Pod
+
+* Log in to the migration pod and verify the buffer settings using the following command:
++
+[source,termina]
+----
+`cat /mnt/extra-v2v-conf/input.conf`
+----
++
+The example output is as follows:
++
+[source,termina]
+----
+VixDiskLib.nfcAio.Session.BufSizeIn64KB=16
+vixDiskLib.nfcAio.Session.BufCount=4
+----
+
+.Enabling Debugging (optional)
+
+* To enable debug logs, convert the configuration to Base64, including a high log level:
++
+[source,termina]
+----
+echo -e
+"`VixDiskLib.nfcAio.Session.BufSizeIn64KB=16\nVixDiskLib.nfcAio.Session.BufCount=4\nVixDiskLib.nfc.LogLevel=4`"
+| base64
+----
++
+[NOTE]
+====
+Adding a high log level will degrade performance and is for debugging purposes only.
+====
+
+[id="mtv-disable-aio-buffer_{context}"]
+== Disabling AIO Buffer Configuration
+
+To disable the AIO buffer configuration, complete the following steps:
+
+* Edit the ForkliftController Object: Remove the previously added lines from the spec section in the ForkliftController object:
++
+[source,termina]
+----
+oc edit forkliftcontroller -n openshift-mtv
+----
+
+* Remove the following lines:
++
+[source,termina]
+----
+virt_v2v_extra_args: "`–vddk-config /mnt/extra-v2v-conf/input.conf`"
+virt_v2v_extra_conf_config_map: "`perf`"
+----
+. Delete the ConfigMap: Remove the perf ConfigMap that was created
+earlier:
+
+`oc delete cm perf -n openshift-mtv`
+
+....
+  3\.	Restart the Forklift Controller Pod (Optional):
+....
+
+If needed, ensure the changes take effect by restarting the forklift-controller pod.
+
+[id="mtv-aio-buffer-key-requirements_{context}"]
+=== Key requirements for AIO Buffer (Asynchronous Buffer Counts) support
+
+.Key requirements for AIO Buffer support
+[width="100%",cols="54%,46%",options="header",]
+|===
+|Version |Support
+
+|vSphere Version
+a|
+vSphere 6.7 Update 3 and later.
+Fully supported in vSphere 7.x versions, including 7.0.x.
+
+|VDDK Version
```

Comment:
this section missing  the next recommendation for VDDK : 

VDDK Version:
Requires VDDK 6.7.3 or newer. The VDDK version typically aligns with the ESXi version installed on the vSphere host.

Reply:
me and daniel made some changes to this section please use the update one : 

VDDK /vSpheres Versions: 

Tested with : 
vSphere : 7.0.3
VDDK : 7.0.3 
* for other  VDDK / vSpheres versions please check the AIO buffer support in the official VMware documentation